unit02/loops.rst

```diff
@@ -0,0 +1,50 @@
+Unit 2 - Loops Learning Objectives
+==================================
+
+Goals
+-----
+
+- Use iteration to repeat the execution of a code block
+- Properly write ``for`` and ``while`` loops
+- Understand which type of loop is better suited for a given situation.
+
+Objectives
+----------
+
+``for`` Loops
+^^^^^^^^^^^^^
+
+Students will be able to:
+
+- Describe the role of each of the 3 components of the ``range`` type:
+  ``start``, ``stop``, ``step``.
+- Recognize that only the ``stop`` component in ``range`` is required.
+- Explain what **zero-based indexing** means, and provide examples.
+- Write a ``for`` loop with ``range(start, stop)`` to iterate upwards
+  from 0 - n, downwards from n - 0, upwards from m - n, and downwards from
+  m - n.
+- Write a ``for`` loop with ``range(start, stop, step)`` to iterate upwards
+  or downwards in increments larger than 1 unit.
+- Iterate over strings without using index values.
+- Iterate over strings using index values.
```

Comment:
We removed this from the chapter.

Reply:
You're right, but for the short term we kept this section in the book.  We'll need to update both after the schools move past chapter 6 this year.

Reply:
I added an issue to the GitHub repo for the LCHS textbook.